src/clientmetricreport/DefaultClientMetricReport.ts

```diff
@@ -575,7 +574,7 @@ export default class DefaultClientMetricReport implements ClientMetricReport {
         const streamId = this.streamMetricReports[ssrc].streamId;
         const attendeeId = streamId
           ? this.videoStreamIndex.attendeeIdForStreamId(streamId)
```

Comment:
Should this be `this.videoStreamIndex?.attendeeIdForStreamId(streamId)` and log some error if this newly added API is used and we get `videoStreamIndex` undefined here.

Reply:
Good point. Resolved in the new revision. Thanks!